### PR TITLE
New uplink item - Secret Storage Crate.

### PR DESCRIPTION
**Pull request by marboww**
## About the PR
This PR adds a stealth box-like crate to the uplink in the Utilities category.

## Why / Balance
This innovation in the game will perfectly allow traitors to conveniently hide their things in an invisible crate (or hide in case of danger) for 2 TC. However, it will not replace the stealth box; 
UPDATE: The syndicate's symbols attract too many eyes, and the crate does not allow entities to pass through

## Technical details
PR takes the syndicate crate as a base and adds two stealth-related components to it.

## Media

https://github.com/space-wizards/space-station-14/assets/152051971/1fcc6eca-6b27-4410-80ab-be648272e830

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### P.S

I am sorry for my English

**Changelog**

- tweak: Price change (from 4 to 2 TC)

### CODE REVIEW

**Comment by icekot8** [comment]
I don't see the point. You can just as easily use a stealth box, but it's 1 TC more expensive, and the box lets entities through while the secret crate doesn't.
I don't find your arguments convincing.

**Comment by Lomovar** [comment]
make it foldable :trollface:

**Comment by marboww** [comment]
> I don't see the point. You can just as easily use a stealth box, but it's 1 TC more expensive, and the box lets entities through while the secret crate doesn't. I don't find your arguments convincing.

In such cases, I can do it cheaper, for example the same 2 TC. Because crates have many disadvantages: syndicate symbolism, etc.  Well, this is my first PR, there are also some mistakes.

**Comment by deltanedas** [comment]
storage implant is 1 more tc and has no non meta counters

**Comment by marboww** [comment]
ok I'll change the price now

**Comment by marboww** [comment]
> I don't see the point. You can just as easily use a stealth box, but it's 1 TC more expensive, and the box lets entities through while the secret crate doesn't. I don't find your arguments convincing.

**Comment by marboww** [comment]
oh damn what have I done...

**Comment by metalgearsloth** [comment]
I don't think this fits, there should be a risk to traitors storing their stuff in maints and this bypasses most of it with an expensive shader.

**Comment by deltanedas** [comment]
no risk just hide it all in toilets

**Comment by marboww** [comment]
> I don't think this fits, there should be a risk to traitors storing their stuff in maints and this bypasses most of it with an expensive shader.

There is still a risk. A person can simply stumble upon an invisible space that does not allow him to pass, and he can calmly open it, call security or scientists.

**Comment by marboww** [comment]
you spend your TK if you want to hide something with a slightly increased chance that it won’t be found, and for only a small price.

File: Resources/Prototypes/Catalog/Fills/Crates/syndicate.yml [hunk 1]
```diff
@@ -7,6 +7,18 @@
...
+  parent: CrateSyndicate
+  name: secret storage crate
+  description: A crate full of riddles...
+  components:
+    - type: Stealth
+      hadOutline: true
+    - type: StealthOnMove
+      passiveVisibilityRate: -0.10
+      movementVisibilityRate: 0.10
+
```
Comment (lzk228) [inline Resources/Prototypes/Catalog/Fills/Crates/syndicate.yml:21]: move just to crates, not filled versions

**Review by lzk228 — CHANGES_REQUESTED** [review]
change id to something nore obvious, like `CrateSyndicateStealth`
also since this, i dont think it needs new name and desc, so just add `suffix: Stealth`

**Comment by marboww** [comment]
> change id to something nore obvious, like `CrateSyndicateStealth`
> also since this, i dont think it needs new name and desc, so just add `suffix: Stealth`

ok

**Comment by marboww** [comment]
hmm, I forgot about the thief, I’d better close PR

**Pull request closed without merging**